pydicom/dataset.py

```diff
@@ -1920,12 +1922,14 @@ def _data_element_from_json(cls, tag, vr, value, value_key,
         elem_value = cls._convert_to_python_number(elem_value, vr)
 
         try:
+            if compat.in_py2 and vr == "PN":
+                elem_value = PersonNameUnicode(elem_value, None)
```

Comment:
yes, this is probably the best way right now to fix this; I was playing with that yesterday, but got distracted...

Reply:
That worked okay, but then hits a test_dataset issue where it expects encodings not to be empty.  Wasn't able to wrap my head around it yet.